### PR TITLE
Update description and tag project metadata.

### PR DESCRIPTION
**Pull request by NTaylorMullen**
#74 

/cc @Eilon @danroth27

### CODE REVIEW

File: src/Microsoft.AspNetCore.Razor.Tools/project.json [hunk 1]
```diff
@@ -1,11 +1,17 @@
...
-  "description": "TagHelper tooling for .NET Core CLI.",
+  "description": "TagHelper tooling for .NET Core CLI. Contains an executable to resolve TagHelperDescriptors for a given project.",
   "packOptions": {
     "repository": {
       "type": "git",
       "url": "git://github.com/AspNet/RazorTooling"
-    }
+    },
+    "tags": [
+      "dotnet-razor-tooling",
```
Comment (Eilon) [inline src/Microsoft.AspNetCore.Razor.Tools/project.json]: This tag isn't saying much... If anything, perhaps say the command line tool name in the description. E.g. `"Contains the dotnet-razor-tooling command used to resolve XYZ for projects at design-time."`.

**Comment by NTaylorMullen** [comment]
:up: :date:

File: src/Microsoft.AspNetCore.Razor.Tools/project.json [hunk 2]
```diff
@@ -1,11 +1,16 @@
...
-  "description": "TagHelper tooling for .NET Core CLI.",
+  "description": "TagHelper tooling for .NET Core CLI. Contains the dotnet-razor-tooling command used to resolve TagHelperDescriptors for projects at design-time.",
   "packOptions": {
     "repository": {
       "type": "git",
       "url": "git://github.com/AspNet/RazorTooling"
-    }
+    },
+    "tags": [
+      "taghelpertooling",
```
Comment (Eilon) [inline src/Microsoft.AspNetCore.Razor.Tools/project.json]: Nix this tag, it's pretty much just repeating the package id. Plus, this tag would clearly never be used on any other package 😄 

Some tags to be inspired by: (i.e. copy the ones that make sense)
* https://github.com/aspnet/Razor/blob/dev/src/Microsoft.AspNetCore.Razor/project.json#L17
* https://github.com/aspnet/Razor/blob/dev/src/Microsoft.AspNetCore.Razor.Runtime/project.json#L18
* https://github.com/aspnet/Mvc/blob/dev/src/Microsoft.AspNetCore.Mvc.TagHelpers/project.json#L9
* https://github.com/aspnet/Mvc/blob/dev/src/Microsoft.AspNetCore.Mvc.Razor/project.json#L9

**Comment by Eilon** [comment]
🕐

**Comment by NTaylorMullen** [comment]
:up: :date: tags :smile:

**Comment by Eilon** [comment]
:shipit:

**Pull request closed without merging**